Formula/biosig.rb

```diff
@@ -31,22 +32,52 @@ class Biosig < Formula
     sha256 "75df4a79b8d3d785942cbfd125ce45de49c3e7fa2cd19adb70caf8c4e30e13f0"
   end
 
+  patch :DATA
+
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
+    system "./configure", "CC=gcc-10", "CXX=g++-10", "--disable-debug",
```

Comment:
We prefer to build with the system compiler. What is missing from clang?

Reply:
linking failed (internally libtinyxml is c++).

Reply:
I do not understand: macOS ships with clang++, which is a C++ compiler. All formulas should be compiled with the system compiler, unless they use a nonstandard GNU C feature which is not supported by clang (nested functions, for example). This needs to be reverted to use clang/clang++, and the linking failure solved.

Reply:
I checked this, the problem occurs when building the java binding for biosig. The error was "ld symbols not found ... clang: error: linker command failed with exit code 1...". Anyway, the language binding of biosig to java does not work in MacOSX (I'm testing on 10.13.6). So this is a non-issue. I've changed the recipe to clang.